core/data/tests/test_datetime_translation/output.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from datetime import datetime
+from pydantic import BaseModel, BeforeValidator, ConfigDict, Field, PlainSerializer
+from typing import Annotated
+
+
+def serialize_datetime_data(utc_time: datetime) -> str:
+        return utc_time.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
once concern that I have here is that there are multiple zulu formats available:
* YYYY-MM-DDTHH:MM:SSZ → 2025-03-13T15:30:45Z (Date + Time + Seconds)
* YYYY-MM-DDTHH:MM:SS.sssZ → 2025-03-13T15:30:45.123Z (Including milliseconds)
* YYYY-MM-DDTHH:MM:SS.ssssssZ → 2025-03-13T15:30:45.123456Z (Including microseconds)
* YYYY-MM-DDTHH:MM:SS.sssssssssZ → 2025-03-13T15:30:45.123456789Z (Including nanoseconds)

and I am wondering if the presumed serialization method (rfc3339) consistently produces this format. Can we check and account for the other formats, otherwise?